Mathlib/Data/Set/Function.lean

```diff
@@ -769,6 +780,22 @@ theorem _root_.Disjoint.image {s t u : Set α} {f : α → β} (h : Disjoint s t
   rw [← hf.image_inter hs ht, h, image_empty]
 #align disjoint.image Disjoint.image
 
+lemma InjOn.image_diff {t : Set α} (h : s.InjOn f) : f '' (s \ t) = f '' s \ f '' (s ∩ t) := by
```

Comment:
and this is the lemma I would consider "auxiliary"
```suggestion
lemma InjOn.image_diff' {t : Set α} (h : s.InjOn f) : f '' (s \ t) = f '' s \ f '' (s ∩ t) := by
```

Reply:
I disagree.